package.json

```diff
@@ -30,7 +32,7 @@
     "build:test": "webpack --config webpack.test.config.js",
     "prestart": "npm run build:dev",
     "start": "grunt start-server-open",
-    "prepublish": "grunt prep-release",
+    "prepublish": "grunt prep-release && webdriver-manager update",
```

Comment:
It's how I guarantee we have the selenium server jar iff we `npm install` from the repo.

https://docs.npmjs.com/misc/scripts
> prepublish: Run BEFORE the package is published. (Also run on local npm install without any arguments.)